frontend/src/components/worksheets/NewUpload/NewUpload.jsx

```diff
@@ -287,6 +289,7 @@ class NewUpload extends React.Component<{
         return (
             <React.Fragment>
                 <input
+                    key={this.props.after_sort_key} // force the NewUpload to update when after_sort_key changes

```

Comment:
What is this change for?

Reply:
without setting the key, previously the props.after_sort_key passed to the NewUpload won't change even if the focused item is changed, which results in the new upload bundles might be inserted into the wrong place.